Modules/_ssl.c

```diff
@@ -51,6 +51,13 @@
             PySSL_BEGIN_ALLOW_THREADS_S(_save);
 #define PySSL_END_ALLOW_THREADS PySSL_END_ALLOW_THREADS_S(_save); }
 
+#ifdef Py_GIL_DISABLED
+#  define PySSL_LOCK(OBJ) 	PyMutex_Lock(&(OBJ)->lock)
```

Comment:
Would it make sense to reuse `PyObject.ob_mutex`, rather than adding a second lock per object?

Reply:
Hmm, I'm not sure. I was under the impression that weird things happened to `ob_mutex` if running inside a critical section. Though, I couldn't find anything else in the stdlib that uses `ob_mutex` manually, I think we're better off just using our own lock (the point of `PyMutex` is to be tiny, anyway).